### PR TITLE
refactor: Fix version, update elvis config and variable renaming

### PR DESCRIPTION
**Pull request by zigge-zagge**
This won't create a release but I think that's better than to let a change like this propagate :)

### CODE REVIEW

**Review by kivra-pauoli — APPROVED** [review]

**Comment by kivra-pauoli** [comment]
I think it's failing because of https://github.com/kivra/id_token/blob/master/Makefile#L36 (head 😖); your the change is sane, though.

Would it be Ok for you to use the `rebar3_lint` plugin for a given version? I think the one you're looking for is https://hex.pm/packages/rebar3_lint/3.0.1.

**Comment by kivra-pauoli** [comment]
@zigge-zagge, we're back in business. Feel free to merge; you have my blessing already.

File: src/id_token_jws.erl [hunk 1]
```diff
@@ -73,11 +73,11 @@ generate_key_for(Alg, Options) ->
           _ -> generate_rsa_key(Alg, Options)
         end,
   Jwk0 = jose_jwk:from_key(Key),
-  Jwk = Jwk0#jose_jwk{fields = #{<<"kid">> => kid(Options),
+  JWK = Jwk0#jose_jwk{fields = #{<<"kid">> => kid(Options),
```
Comment (zigge-zagge) [inline src/id_token_jws.erl:76]: So this was needed for the `consistent_variable_casing` rule? Looks a bit off with `Jwk0` and `JWK`.

**Review by zigge-zagge — COMMENTED** [review]
Looks good!

File: src/id_token_jws.erl (hunk 1 above)
Comment (kivra-pauoli) [reply src/id_token_jws.erl:76]: I can "fix" that. What the rule check is "lowercase comparison" of variables in the whole file. `JWk`, `JWK`, `JwK`; it doesn't care about the `0` (oh, how I miss Elixir's `|` operator)…

Comment (kivra-pauoli) [reply src/id_token_jws.erl:76]: I'll move all `Jw`... wait a min.

Comment (zigge-zagge) [reply src/id_token_jws.erl]: My point is that it should also complain about `Jwk0` so that we'd have to change it `JWK0`, right?

Comment (zigge-zagge) [reply src/id_token_jws.erl]: We can still just do that though.

File: elvis.config
Comment (kivra-pauoli) [inline elvis.config:1]: All this means the `ruleset` rules are applied as default. There's a new ruleset (`strict`) that we can try in the future (that includes more stricter default rules).

File: src/id_token_jws.erl (hunk 1 above)
Comment (zigge-zagge) [reply src/id_token_jws.erl]: But now I see. With that `0` it's a totally different variable name according to Elvis. Not really how we see it though.

**Pull request merged**